### ISSUE TITLE
Warning: Unknown prop attr on tag

### ISSUE THREAD

**Issue by witcradg** [react-icons/react-icons#172]
bundle.js:8939 Warning: Unknown prop `attr` on <svg> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in svg (created by IconBase)
    in IconBase (created by Unknown)
    in Unknown
    in div

version:     "react-icons": "^3.0.5"
Chrome Version 68.0.3440.84 (Official Build) (64-bit)
using Babel and WebPack

I've stripped everything out of the project except these lines in index.js:
```
import React from 'react';
import { render } from 'react-dom';
import { FaBeer } from 'react-icons/fa';

window.React = React;

render( 
	<div>
        	< FaBeer />
	</div>,
	document.getElementById('react-container')
)
```
The icon displays properly but I don't understand what the issue is that's producing this warning.
I'm new to React and react-icons.

**Comment by kamijin-fanta** [comment]
I understand this problem.
we will need fix.

**Comment by ahutchings** [comment]
@kamijin-fanta @witcradg This was addressed in #184 which is in the [v3.1.0 release](https://github.com/react-icons/react-icons/releases/tag/v3.1.0).

**Comment by kamijin-fanta** [comment]
it resolved!
